cypress/e2e/app_tests/testsHW12/step-by-stepTextChecking.cy.js

```diff
@@ -0,0 +1,28 @@
+///<reference types="cypress" />
+
+//const { Button } = require("bootstrap");
+
+describe("tests HW12 suite", () => {
+  it("step-by-stepTextChecking", () => {
+    cy.visit("pages/layout/stepper");
+
+    cy.get(".lorem.ng-star-inserted").contains("Proin varius");
+    cy.get(
+      ".appearance-filled.size-medium.status-primary.shape-rectangle.ng-star-inserted.transitions",
+    )
```

Comment:
This selector can be shorter
```suggestion
    cy.get('.vertical ').contains('next').click()
```